yellowbrick/base.py

```diff
@@ -407,3 +408,119 @@ def predict(self, X, y):
         """
         for model in self.models:
             yield cvp(model, X, y, cv=12)
+
+
+
+#TODO it appears that the previous class (MultiModelMixin) is not actually used
+# anywhere and would be redundant to this new class. Remove that one?
+class MultipleVisualizer(Visualizer):
+    """
+    Used as a base class for visualizers that use subplots.
+
+    Parameters
+    ----------
+    visualizers : A list of instantiated visualizers
+
+    nrows: integer, default: None
+        The number of rows desired, if you would like a fixed number of rows.
+        Specify only one of nrows and ncols, the other should be None. If you 
+        specify nrows, there will be enough columns created to fit all the 
+        visualizers specified in the visualizers list. 
+    
+    ncols: integer, default: None
+        The number of columns desired, if you would like a fixed number of columns.
+        Specify only one of nrows and ncols, the other should be None. If you 
+        specify ncols, there will be enough rows created to fit all the 
+        visualizers specified in the visualizers list. 
+
+    axarr: matplotlib.axarr, default: None.
+        If you want to put the plot onto an existing axarr, specify it here. Otherwise a new
+        one will be created. 
+
+    kwargs : additional keyword arguments, default: None
+        Any additional keyword arguments will be passed on to the fit() method and therefore 
+        passed on to the fit() method of the wrapped estimators, if applicable. Otherwise ignored.
+
+    Examples
+    --------
+    >>> from yellowbrick.base import MultipleVisualizer
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from yellowbrick.classifier import ConfusionMatrix
+    >>> from yellowbrick.classifier import ClassBalance
+    >>> model = LogisticRegression()
+    >>> visualizers = [ClassBalance(model),ConfusionMatrix(model)]
+    >>> mv = MultipleVisualizer(visualizers, ncols=2)
+    >>> mv.fit(X_train, y_train)
+    >>> mv.score(X_test, y_test)
+    >>> mv.poof()
+    """
+    def __init__(self, visualizers = [], nrows = None, ncols = None, axarr = None, **kwargs):
+        self.visualizers = visualizers
+        self.plotcount = len(visualizers)
+        if nrows == None and ncols == None:
+            self.ncols = 1
+            self.nrows = self.plotcount
+        elif ncols == None:
+            self.nrows = nrows
+            self.ncols = math.ceil(self.plotcount / self.nrows)
+        elif nrows == None:
+            self.ncols = ncols
+            self.nrows = math.ceil(self.plotcount / self.ncols)
+        else:
+            raise YellowbrickValueError("You can only specify either nrows or ncols, \
+                the other will be calculated based on the length of the list of visualizers.")
+        
+
+        if axarr == None:
+            fig, axarr = plt.subplots(self.nrows, self.ncols, squeeze = False)
+        
+        self.axarr = axarr
```

Comment:
What happens when a user accesses `MultipleVisualzer.ax`? Right now I believe what would happen is that the current axes in the `axarr` would be fetched and stored as `_ax`; however, I believe correct behavior would be akin to `get_current_axes`. I recommend adding the following:

```python 
@property 
def ax(self):
     """
     Override Visualizer.ax to return the current axis 
     """
     return plt.gca() 

@ax.setter
def ax(self, ax):
     raise YellowbrickTypeError("cannot set new axes objects on multiple visualizers")
```

Reply:
Done - I didn't test previous behavior, but adding this seems like it would override any such behavior and does what we want it to whether or not previous behavior was problematic.